lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -2860,15 +2862,6 @@ void mme_app_handle_create_dedicated_bearer_rej(
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
 
-//------------------------------------------------------------------------------
```

Comment:
I believe this function is completely unreferenced dead code, do we want to leave a commented TODO and drop the function?